app/src/main/res/values/strings.xml

```diff
@@ -104,6 +104,8 @@
     <string name="settings_key_display_barcode_max_brightness" translatable="false">pref_display_card_max_brightness</string>
     <string name="settings_lock_barcode_orientation">Lock barcode orientation</string>
     <string name="settings_key_lock_barcode_orientation" translatable="false">pref_lock_barcode_orientation</string>
+    <string name="settings_keep_screen_on">Keep screen on</string>
+    <string name="settings_key_keep_screen_on" translatable="false">pref_keep_screen_on</string>
 
```

Comment:
If we add another option for the lock screen stuff, may I suggest "Show loyalty card over lock screen", similar to how Transportr does it?

I can figure out the second row thing myself though, don't worry about that for now.

![image](https://user-images.githubusercontent.com/1885159/108118299-6ebdac80-709e-11eb-8cfa-713b5da7bb0f.png)